examples/belief_propagation/ApproximateNormsV2.jl

```diff
@@ -0,0 +1,140 @@
+using Compat
+using ITensors
+using Metis
+using ITensorNetworks
+using Random
+using NamedGraphs: add_vertex!, add_edge!
+using AbstractTrees
+using CairoMakie
+using Permutations
+using Statistics
+
+using ITensorNetworks:
+  compute_message_tensors,
+  calculate_contraction,
+  group_partition_vertices,
+  contract_inner,
+  split_index,
+  ising_network,
+  get_environment,
+  find_subgraph,
+  update_all_mts
+
+using OMEinsumContractionOrders
+include("ContractionTreeFunctions.jl")
```

Comment:
I don't see this file included in the PR.

Reply:
Maybe this whole file should be left for a future PR?

Reply:
Yes sorry it wasn't supposed to be included. I accidentally added it and the code-workspace file. Will take them out with .gitignore

Reply:
You shouldn't add a normal file like that to `.gitignore`, that's only for files you know you will always want to ignore (like system files, ones generated by text editors, etc.). You should be able to just remove this one with `git rm`, and just not include it with `git add` in the first place if you're not ready to commit it.